tests/conftest.py

```diff
@@ -101,7 +106,8 @@ def supply():
 def game(player):
     game = Game(
         players=[player],
-        expansions=[base_set],
+        expansions=[base_set, intrigue_set],
+        kingdom_cards=[mill, smithy],
```

Comment:
Are mill and smithy required here (and line 126)? I could have overlooked something.

Reply:
I'm used them in some of the tests for cards that interact with the supply. I used mill in `test_ironworks,py` and `test_replace.py` and smithy in `test_ironworks.py` and `test_lurker.py`.

Reply:
Ok thanks for the explanation, good to keep them in then.